### PR TITLE
[test_auto_negotiation] skip an test for boradcom platform

### PR DESCRIPTION
**Pull request by StormLiangMS**
### Description of PR

auto negotiation test doesn't work for broadcom platform, skip.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -299,3 +299,12 @@ macsec/test_macsec.py:
     reason: "This test can only run on 7280 CR3"
     conditions:
       - "platform not in ['x86_64-arista_7280cr3mk_32d4', 'x86_64-kvm_x86_64-r0']"
+
+#######################################
+#####        auto negotiation     #####
+#######################################
+platform_tests/test_auto_negotiation.py:
```
[the remark this answers is not in the document]
Comment (StormLiangMS) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: Sure, changed the macsec one too.

**Review by wangxin — APPROVED** [review]

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -154,6 +163,16 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      reason: "Forward action not supported in cisco-8000"
      conditions:
         - "asic_type in ['cisco-8000']"
+
+#######################################
+#####        auto negotiation     #####
```
Comment (wangxin) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:168]: It would be more consistent if the comment here can be `platform_tests`.

Comment (StormLiangMS) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: sure

**Review by yxieca — DISMISSED** [review]

**Comment by StormLiangMS** [comment]
after rebase, there are many differences, to use a new PR to commit, close this one. 
https://github.com/Azure/sonic-mgmt/pull/5467/files

**Pull request closed without merging**